### ISSUE TITLE
Copy of  2.1.1 Popup Tabs- Student- Accessibility Fixes - Level A - Keyboard

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#3115]
**User Story**
**As a student, I need 

**Context:**
- Theses  issues were uncovered through accessibility audit via CITZ. 
- Detailed Spreadsheet for Student s inserted inline and also attached here: [2024_Student Application_Audit.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0xsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--578821d4addd12c1babcafba5c861e1f5456bc9c/2024_Student%20Application_Audit.xlsx)
- All functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

**The Problem**
- Users should be able to tab into any popup (alert, warning, notifications) to dismiss the information on their own accord without disrupting the flow of focus.

**Student** 
Adjust the following Accessibility Criteria for the following  pages 
- [ ]  Login 1 and 2
- [ ]  Profile 
- [ ] Home page 
- [ ] Start new application 
- [ ] My applications 
- [ ] File uploader 
- [ ] Request a change 
- [ ] Overawards balance 

**Acceptance Criteria**
-[ ] In the Student Portal (pages listed above) ensure all elements that can be interacted with by mouse or trackpad can be interacted with via keyboard.

**Comment by HRAGANBC** [comment]
@ninosamson 
Testing identified that Space Bar can be used to select check-boxes throughout the system, and keyboard can be used to Tab into the box. 

Is using Space bar sufficient for 2.0? Can I 2.X this and make common checkbox component able to accomodate Enter?
